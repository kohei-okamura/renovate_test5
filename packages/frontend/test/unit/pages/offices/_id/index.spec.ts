/*
 * Copyright © 2019 EUSTYLE LABORATORY - ALL RIGHTS RESERVED.
 * UNAUTHORIZED COPYING OF THIS FILE, VIA ANY MEDIUM IS STRICTLY PROHIBITED PROPRIETARY AND CONFIDENTIAL.
 */
import { reactive } from '@nuxtjs/composition-api'
import { MountOptions, Wrapper } from '@vue/test-utils'
import { OfficeQualification } from '@zinger/enums/lib/office-qualification'
import { Permission } from '@zinger/enums/lib/permission'
import { Purpose } from '@zinger/enums/lib/purpose'
import { createMock } from '@zinger/helpers/testing/create-mock'
import Vue from 'vue'
import { dwsAreaGradesStateKey, dwsAreaGradesStoreKey } from '~/composables/stores/use-dws-area-grades-store'
import { ltcsAreaGradesStateKey, ltcsAreaGradesStoreKey } from '~/composables/stores/use-ltcs-area-grades-store'
import { officeStateKey } from '~/composables/stores/use-office-store'
import { sessionStoreKey } from '~/composables/stores/use-session-store'
import { Auth } from '~/models/auth'
import OfficesViewPage from '~/pages/offices/_id/index.vue'
import { createTabService, TabService } from '~/services/tab-service'
import { createDwsAreaGradeStubs } from '~~/stubs/create-dws-area-grade-stub'
import { createDwsAreaGradesStoreStub } from '~~/stubs/create-dws-area-grades-store-stub'
import { createLtcsAreaGradeStubs } from '~~/stubs/create-ltcs-area-grade-stub'
import { createLtcsAreaGradesStoreStub } from '~~/stubs/create-ltcs-area-grades-store-stub'
import { createOfficeResponseStub } from '~~/stubs/create-office-response-stub'
import { createOfficeStoreStub } from '~~/stubs/create-office-store-stub'
import { createOfficeStubs } from '~~/stubs/create-office-stub'
import { createAuthStub } from '~~/test/helpers/create-auth-stub'
import { provides } from '~~/test/helpers/provides'
import { setData } from '~~/test/helpers/set-data'
import { setupComponentTest } from '~~/test/helpers/setup-component-test'

describe('pages/offices/_id/index.vue', () => {
  const { mount } = setupComponentTest()
  const stub = createOfficeStubs().find(x => {
    return x.purpose === Purpose.internal && x.qualifications.includes(OfficeQualification.dwsHomeHelpService)
  })!
  const id = stub.id
  const $tabs = createMock<TabService>({
    ...createTabService()
  })
  let wrapper: Wrapper<Vue & any>

  function mountComponent (
    options: MountOptions<Vue> = {},
    auth: Partial<Auth> = { isSystemAdmin: true }
  ) {
    const dwsAreaGradesStore = createDwsAreaGradesStoreStub({
      dwsAreaGrades: createDwsAreaGradeStubs()
    })
    const ltcsAreaGradesStore = createLtcsAreaGradesStoreStub({
      ltcsAreaGrades: createLtcsAreaGradeStubs()
    })
    const response = createOfficeResponseStub(id)
    const store = createOfficeStoreStub(response)
    wrapper = mount(OfficesViewPage, {
      ...options,
      ...provides(
        [dwsAreaGradesStoreKey, dwsAreaGradesStore],
        [dwsAreaGradesStateKey, dwsAreaGradesStore.state],
        [ltcsAreaGradesStoreKey, ltcsAreaGradesStore],
        [ltcsAreaGradesStateKey, ltcsAreaGradesStore.state],
        [officeStateKey, store.state],
        [sessionStoreKey, createAuthStub(auth)]
      )
    })
  }

  function unmountComponent () {
    wrapper.destroy()
  }

  afterEach(() => {
    jest.clearAllMocks()
  })

  it('should be rendered correctly', () => {
    mountComponent({ mocks: { $tabs } })
    expect(wrapper).toMatchSnapshot()
    unmountComponent()
  })

  describe('tabs', () => {
    beforeAll(() => {
      mountComponent({ mocks: { $tabs } })
    })

    afterAll(() => {
      unmountComponent()
    })

    it('should have correct tab items', () => {
      const expected = ([
        reactive({ label: '基本情報', to: '#office' }),
        reactive({ label: '算定情報', to: '#calc-specs' })
      ])
      const tabs = wrapper.vm.$data.tabs

      expect(tabs).toStrictEqual(expected)
    })

    it('should not have calc-spec tab item if purpose is external', async () => {
      const expected = ([
        reactive({ label: '基本情報', to: '#office' })
      ])
      await setData(wrapper, { office: { purpose: Purpose.external } })
      const tabs = wrapper.vm.$data.tabs

      expect(tabs).toStrictEqual(expected)
    })

    it('should display "office" tab in initial state', () => {
      const tabItem = wrapper.find('[value="office"]')
      expect(tabItem.isVisible()).toBeTrue()
    })
  })

  describe.each([
    ['office'],
    ['calc-specs']
  ])('FAB (speed dial) in "%s" tab', name => {
    const requiredPermission: Permission[] = [Permission.updateInternalOffices, Permission.updateExternalOffices]

    it('should be rendered when session auth is system admin', () => {
      mountComponent({ mocks: { $tabs } })
      expect(wrapper).toContainElement(`[value="${name}"] [data-fab]`)
      expect(wrapper.find(`[value="${name}"] [data-fab]`)).toMatchSnapshot()
      unmountComponent()
    })

    it(`should be rendered when the staff has permissions: ${requiredPermission}`, () => {
      const permissions = requiredPermission
      mountComponent({ mocks: { $tabs } }, { permissions })
      expect(wrapper).toContainElement(`[value="${name}"] [data-fab]`)
      expect(wrapper.find(`[value="${name}"] [data-fab]`)).toMatchSnapshot()
      unmountComponent()
    })

    it(`should not be rendered when the staff does not have permissions: ${requiredPermission}`, () => {
      const permissions = Permission.values.filter(x => !requiredPermission.includes(x))
      mountComponent({ mocks: { $tabs } }, { permissions })
      expect(wrapper).not.toContainElement(`[value="${name}"] [data-fab]`)
      unmountComponent()
    })
  })
})
