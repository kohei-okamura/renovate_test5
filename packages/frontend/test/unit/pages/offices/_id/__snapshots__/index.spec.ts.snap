// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/offices/_id/index.vue FAB (speed dial) in "calc-specs" tab should be rendered when session auth is system admin 1`] = `
<z-fab-speed-dial-stub
  data-fab=""
  icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
>
  <z-fab-speed-dial-button-stub
    color="secondary"
    icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
    to="/offices/2/home-visit-long-term-care-calc-specs/new"
  >
          算定情報（介保・訪問介護）を登録
  </z-fab-speed-dial-button-stub>
  <z-fab-speed-dial-button-stub
    color="secondary"
    icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
    to="/offices/2/visiting-care-for-pwsd-calc-specs/new"
  >
          算定情報（障害・重度訪問介護）を登録
  </z-fab-speed-dial-button-stub>
  <z-fab-speed-dial-button-stub
    color="secondary"
    icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
    to="/offices/2/home-help-service-calc-specs/new"
  >
          算定情報（障害・居宅介護）を登録
  </z-fab-speed-dial-button-stub>
</z-fab-speed-dial-stub>
`;

exports[`pages/offices/_id/index.vue FAB (speed dial) in "calc-specs" tab should be rendered when the staff has permissions: offices/update,external-offices/update 1`] = `
<z-fab-speed-dial-stub
  data-fab=""
  icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
>
  <z-fab-speed-dial-button-stub
    color="secondary"
    icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
    to="/offices/2/home-visit-long-term-care-calc-specs/new"
  >
          算定情報（介保・訪問介護）を登録
  </z-fab-speed-dial-button-stub>
  <z-fab-speed-dial-button-stub
    color="secondary"
    icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
    to="/offices/2/visiting-care-for-pwsd-calc-specs/new"
  >
          算定情報（障害・重度訪問介護）を登録
  </z-fab-speed-dial-button-stub>
  <z-fab-speed-dial-button-stub
    color="secondary"
    icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
    to="/offices/2/home-help-service-calc-specs/new"
  >
          算定情報（障害・居宅介護）を登録
  </z-fab-speed-dial-button-stub>
</z-fab-speed-dial-stub>
`;

exports[`pages/offices/_id/index.vue FAB (speed dial) in "office" tab should be rendered when session auth is system admin 1`] = `
<a
  bottom=""
  class="z-fab-stub"
  data-fab=""
  fixed=""
  icon="M20.71,7.04C21.1,6.65 21.1,6 20.71,5.63L18.37,3.29C18,2.9 17.35,2.9 16.96,3.29L15.12,5.12L18.87,8.87M3,17.25V21H6.75L17.81,9.93L14.06,6.18L3,17.25Z"
  nuxt=""
  right=""
  to="/offices/2/edit"
>
        事業所情報を編集
</a>
`;

exports[`pages/offices/_id/index.vue FAB (speed dial) in "office" tab should be rendered when the staff has permissions: offices/update,external-offices/update 1`] = `
<a
  bottom=""
  class="z-fab-stub"
  data-fab=""
  fixed=""
  icon="M20.71,7.04C21.1,6.65 21.1,6 20.71,5.63L18.37,3.29C18,2.9 17.35,2.9 16.96,3.29L15.12,5.12L18.87,8.87M3,17.25V21H6.75L17.81,9.93L14.06,6.18L3,17.25Z"
  nuxt=""
  right=""
  to="/offices/2/edit"
>
        事業所情報を編集
</a>
`;

exports[`pages/offices/_id/index.vue should be rendered correctly 1`] = `
<div
  breadcrumbs="[object Object],[object Object]"
  class="z-page-stub"
  compact="compact"
  tabs="[object Object],[object Object]"
>
  <div
    class="z-page-stub-slot-tabs"
  />
  <div
    class="z-page-stub-slot-title"
  />
  <div
    class="v-window adjustEdge transparent v-item-group theme--light v-tabs-items"
  >
    <div
      class="v-window__container"
    >
      <v-tab-item-stub
        value="office"
      >
        <div
          class="z-data-card-stub"
          title="基本情報"
        >
          <div
            class="z-data-card-stub-slot-actions"
          />
          <z-data-card-item-stub
            icon="M11.19,2.25C10.93,2.25 10.67,2.31 10.42,2.4L3.06,5.45C2.04,5.87 1.55,7.04 1.97,8.05L6.93,20C7.24,20.77 7.97,21.23 8.74,21.25C9,21.25 9.27,21.22 9.53,21.1L16.9,18.05C17.65,17.74 18.11,17 18.13,16.25C18.14,16 18.09,15.71 18,15.45L13,3.5C12.71,2.73 11.97,2.26 11.19,2.25M14.67,2.25L18.12,10.6V4.25A2,2 0 0,0 16.12,2.25M20.13,3.79V12.82L22.56,6.96C22.97,5.94 22.5,4.78 21.47,4.36M11.19,4.22L16.17,16.24L8.78,19.3L3.8,7.29"
            label="事業者区分"
            value="自社"
          />
          <z-data-card-item-stub
            icon="M18,15H16V17H18M18,11H16V13H18M20,19H12V17H14V15H12V13H14V11H12V9H20M10,7H8V5H10M10,11H8V9H10M10,15H8V13H10M10,19H8V17H10M6,7H4V5H6M6,11H4V9H6M6,15H4V13H6M6,19H4V17H6M12,7V3H2V21H22V7H12Z"
            label="事業所名"
            value="土屋訪問介護事業所 仙台"
          />
          <z-data-card-item-stub
            label="事業所名：フリガナ"
            value="ツチヤホウモンカイゴジギョウショセンダイ"
          />
          <z-data-card-item-stub
            label="事業所名：略称"
            value="仙台"
          />
          <z-data-card-item-stub
            label="事業所グループ"
            value="福島"
          />
          <z-data-card-item-stub
            icon="M12,2A10,10 0 0,1 22,12A10,10 0 0,1 12,22A10,10 0 0,1 2,12A10,10 0 0,1 12,2M12,4A8,8 0 0,0 4,12A8,8 0 0,0 12,20A8,8 0 0,0 20,12A8,8 0 0,0 12,4M12,10.5A1.5,1.5 0 0,1 13.5,12A1.5,1.5 0 0,1 12,13.5A1.5,1.5 0 0,1 10.5,12A1.5,1.5 0 0,1 12,10.5M7.5,10.5A1.5,1.5 0 0,1 9,12A1.5,1.5 0 0,1 7.5,13.5A1.5,1.5 0 0,1 6,12A1.5,1.5 0 0,1 7.5,10.5M16.5,10.5A1.5,1.5 0 0,1 18,12A1.5,1.5 0 0,1 16.5,13.5A1.5,1.5 0 0,1 15,12A1.5,1.5 0 0,1 16.5,10.5Z"
            label="状態"
            value="準備中"
          />
          <z-data-card-item-stub
            icon="M12,6.5A2.5,2.5 0 0,1 14.5,9A2.5,2.5 0 0,1 12,11.5A2.5,2.5 0 0,1 9.5,9A2.5,2.5 0 0,1 12,6.5M12,2A7,7 0 0,1 19,9C19,14.25 12,22 12,22C12,22 5,14.25 5,9A7,7 0 0,1 12,2M12,4A5,5 0 0,0 7,9C7,10 7,12 12,18.71C17,12 17,10 17,9A5,5 0 0,0 12,4Z"
            label="住所"
          >
          〒759-6524
            <br />
          山口県下関市吉見上2-16-17
            <!---->
          </z-data-card-item-stub>
          <z-data-card-item-stub
            icon="M6.62,10.79C8.06,13.62 10.38,15.94 13.21,17.38L15.41,15.18C15.69,14.9 16.08,14.82 16.43,14.93C17.55,15.3 18.75,15.5 20,15.5A1,1 0 0,1 21,16.5V20A1,1 0 0,1 20,21A17,17 0 0,1 3,4A1,1 0 0,1 4,3H7.5A1,1 0 0,1 8.5,4C8.5,5.25 8.7,6.45 9.07,7.57C9.18,7.92 9.1,8.31 8.82,8.59L6.62,10.79Z"
            label="電話番号"
            value="0550-08-3091"
          />
          <z-data-card-item-stub
            label="FAX番号"
            value="-"
          />
          <z-data-card-item-stub
            icon="M22 6C22 4.9 21.1 4 20 4H4C2.9 4 2 4.9 2 6V18C2 19.1 2.9 20 4 20H20C21.1 20 22 19.1 22 18V6M20 6L12 11L4 6H20M20 18H4V8L12 13L20 8V18Z"
            label="メールアドレス"
          >
            <a
              href="mailto:byyjgonh5248@vhl.example3.co.jp"
            >
              byyjgonh5248@vhl.example3.co.jp
            </a>
          </z-data-card-item-stub>
          <z-data-card-item-stub
            label="指定区分"
          >
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              居宅介護（障害福祉サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              重度訪問介護（障害福祉サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              地域生活支援事業・移動支援（障害福祉サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              その他障害福祉サービス
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              訪問介護（介護保険サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              総合事業・訪問型サービス（介護保険サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              居宅介護支援（介護保険サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              介護予防支援（介護保険サービス）
            </v-chip-stub>
            <v-chip-stub
              active="true"
              activeclass=""
              closeicon="$delete"
              closelabel="$vuetify.close"
              filtericon="$complete"
              label="true"
              small="true"
              tag="span"
            >
              その他介護保険サービス
            </v-chip-stub>
          </z-data-card-item-stub>
          <div
            class="z-data-card-stub-slot-header"
          />
        </div>
        <!---->
        <!---->
        <div
          class="z-data-card-stub"
          title="介護保険サービス：訪問介護"
        >
          <div
            class="z-data-card-stub-slot-actions"
          />
          <z-data-card-item-stub
            icon="M18 12.24V22H17.06V12.24C17.06 12.09 17 12 16.93 11.89C16.84 11.8 16.74 11.76 16.62 11.76C16.47 11.76 16.36 11.8 16.27 11.89C16.18 12 16.14 12.1 16.14 12.24V13.16H15.23V12.5C14.53 12.33 13.9 12.04 13.35 11.63C12.8 11.22 12.34 10.74 11.96 10.19L11.61 11.39C11.5 11.81 11.5 12.24 11.5 12.68L11.5 13L11.5 13.33L13.35 15.94V22H11.5V17.34L9.82 15L9.65 18.25L6.86 22L5.38 20.87L7.77 17.64V12.68C7.77 12.15 7.82 11.63 7.91 11.11L8.25 9.54L6.86 10.32V13.63H5V9.23L10 6.4C10.29 6.26 10.59 6.18 10.91 6.18C11.23 6.18 11.54 6.27 11.83 6.44C12.15 6.62 12.39 6.88 12.57 7.23L13.31 8.8C13.6 9.38 14.04 9.87 14.64 10.26C15.23 10.65 15.89 10.85 16.62 10.85C17 10.85 17.32 11 17.6 11.24C17.88 11.5 18 11.83 18 12.24M12 2C13.11 2 14 2.9 14 4C14 5.11 13.11 6 12 6C10.9 6 10 5.11 10 4C10 2.9 10.9 2 12 2Z"
            label="事業所番号"
            value="4072232252"
          />
          <z-data-card-item-stub
            label="開設日"
          >
            <time
              datetime="1987-09-20"
            >
              昭和62年9月20日
            </time>
          </z-data-card-item-stub>
          <z-data-card-item-stub
            label="指定更新期日"
          >
            <time
              datetime="1984-08-30"
            >
              昭和59年8月30日
            </time>
          </z-data-card-item-stub>
          <z-data-card-item-stub
            label="地域区分"
            value="3級地"
          />
          <div
            class="z-data-card-stub-slot-header"
          />
        </div>
        <div
          class="z-data-card-stub"
          title="介護保険サービス：総合事業・訪問型サービス"
        >
          <div
            class="z-data-card-stub-slot-actions"
          />
          <z-data-card-item-stub
            icon="M18 12.24V22H17.06V12.24C17.06 12.09 17 12 16.93 11.89C16.84 11.8 16.74 11.76 16.62 11.76C16.47 11.76 16.36 11.8 16.27 11.89C16.18 12 16.14 12.1 16.14 12.24V13.16H15.23V12.5C14.53 12.33 13.9 12.04 13.35 11.63C12.8 11.22 12.34 10.74 11.96 10.19L11.61 11.39C11.5 11.81 11.5 12.24 11.5 12.68L11.5 13L11.5 13.33L13.35 15.94V22H11.5V17.34L9.82 15L9.65 18.25L6.86 22L5.38 20.87L7.77 17.64V12.68C7.77 12.15 7.82 11.63 7.91 11.11L8.25 9.54L6.86 10.32V13.63H5V9.23L10 6.4C10.29 6.26 10.59 6.18 10.91 6.18C11.23 6.18 11.54 6.27 11.83 6.44C12.15 6.62 12.39 6.88 12.57 7.23L13.31 8.8C13.6 9.38 14.04 9.87 14.64 10.26C15.23 10.65 15.89 10.85 16.62 10.85C17 10.85 17.32 11 17.6 11.24C17.88 11.5 18 11.83 18 12.24M12 2C13.11 2 14 2.9 14 4C14 5.11 13.11 6 12 6C10.9 6 10 5.11 10 4C10 2.9 10.9 2 12 2Z"
            label="事業所番号"
            value="7964911639"
          />
          <z-data-card-item-stub
            label="開設日"
          >
            <time
              datetime="2008-09-10"
            >
              平成20年9月10日
            </time>
          </z-data-card-item-stub>
          <z-data-card-item-stub
            label="指定更新期日"
          >
            <time
              datetime="2004-07-13"
            >
              平成16年7月13日
            </time>
          </z-data-card-item-stub>
          <div
            class="z-data-card-stub-slot-header"
          />
        </div>
        <div
          class="z-data-card-stub"
          title="介護保険サービス：居宅介護支援"
        >
          <div
            class="z-data-card-stub-slot-actions"
          />
          <z-data-card-item-stub
            icon="M18 12.24V22H17.06V12.24C17.06 12.09 17 12 16.93 11.89C16.84 11.8 16.74 11.76 16.62 11.76C16.47 11.76 16.36 11.8 16.27 11.89C16.18 12 16.14 12.1 16.14 12.24V13.16H15.23V12.5C14.53 12.33 13.9 12.04 13.35 11.63C12.8 11.22 12.34 10.74 11.96 10.19L11.61 11.39C11.5 11.81 11.5 12.24 11.5 12.68L11.5 13L11.5 13.33L13.35 15.94V22H11.5V17.34L9.82 15L9.65 18.25L6.86 22L5.38 20.87L7.77 17.64V12.68C7.77 12.15 7.82 11.63 7.91 11.11L8.25 9.54L6.86 10.32V13.63H5V9.23L10 6.4C10.29 6.26 10.59 6.18 10.91 6.18C11.23 6.18 11.54 6.27 11.83 6.44C12.15 6.62 12.39 6.88 12.57 7.23L13.31 8.8C13.6 9.38 14.04 9.87 14.64 10.26C15.23 10.65 15.89 10.85 16.62 10.85C17 10.85 17.32 11 17.6 11.24C17.88 11.5 18 11.83 18 12.24M12 2C13.11 2 14 2.9 14 4C14 5.11 13.11 6 12 6C10.9 6 10 5.11 10 4C10 2.9 10.9 2 12 2Z"
            label="事業所番号"
            value="5742599976"
          />
          <z-data-card-item-stub
            label="開設日"
          >
            <time
              datetime="2002-02-13"
            >
              平成14年2月13日
            </time>
          </z-data-card-item-stub>
          <z-data-card-item-stub
            label="指定更新期日"
          >
            <time
              datetime="2019-03-04"
            >
              平成31年3月4日
            </time>
          </z-data-card-item-stub>
          <z-data-card-item-stub
            label="地域区分"
            value="1級地"
          />
          <div
            class="z-data-card-stub-slot-header"
          />
        </div>
        <!---->
        <z-system-meta-card-stub
          createdat="1973-09-20T02:29:06+0900"
          id="2"
          updatedat="1972-11-03T20:17:27+0900"
        />
        <a
          bottom=""
          class="z-fab-stub"
          data-fab=""
          fixed=""
          icon="M20.71,7.04C21.1,6.65 21.1,6 20.71,5.63L18.37,3.29C18,2.9 17.35,2.9 16.96,3.29L15.12,5.12L18.87,8.87M3,17.25V21H6.75L17.81,9.93L14.06,6.18L3,17.25Z"
          nuxt=""
          right=""
          to="/offices/2/edit"
        >
        事業所情報を編集
        </a>
      </v-tab-item-stub>
      <v-tab-item-stub
        value="calc-specs"
      >
        <div
          class="z-home-help-service-calc-specs-card root"
          data-z-data-accordion=""
        >
          <z-subheader-stub>
            算定情報（障害・居宅介護）
          </z-subheader-stub>
          <div
            class="v-item-group theme--light v-expansion-panels"
          >
            <div
              aria-expanded="false"
              class="v-expansion-panel panel"
              data-expansion-panel=""
            >
              <button
                class="v-expansion-panel-header header"
                data-expansion-panel-header=""
                type="button"
              >
                <div
                  class="item"
                >
                  <div
                    class="text-caption text--secondary"
                  >
            適用期間
                  </div>
                  <div
                    class="text-body-2 text--primary"
                  >
                    <time
                      datetime="2008-04-16"
                    >
                      平成20年4月16日
                    </time>
                    <span>
                      〜
                    </span>
                    <time
                      datetime="2012-10-29"
                    >
                      平成24年10月29日
                    </time>
                  </div>
                </div>
                <transition-stub
                  name="fade-transition"
                >
                  <div
                    class="v-expansion-panel-header__icon"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-chevron-down theme--light"
                    />
                  </div>
                </transition-stub>
              </button>
              <transition-stub
                name="expand-transition"
              >
                <!---->
              </transition-stub>
            </div>
            <div
              aria-expanded="false"
              class="v-expansion-panel panel"
              data-expansion-panel=""
            >
              <button
                class="v-expansion-panel-header header"
                data-expansion-panel-header=""
                type="button"
              >
                <div
                  class="item"
                >
                  <div
                    class="text-caption text--secondary"
                  >
            適用期間
                  </div>
                  <div
                    class="text-body-2 text--primary"
                  >
                    <time
                      datetime="1972-05-03"
                    >
                      昭和47年5月3日
                    </time>
                    <span>
                      〜
                    </span>
                    <time
                      datetime="1989-04-02"
                    >
                      平成1年4月2日
                    </time>
                  </div>
                </div>
                <transition-stub
                  name="fade-transition"
                >
                  <div
                    class="v-expansion-panel-header__icon"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-chevron-down theme--light"
                    />
                  </div>
                </transition-stub>
              </button>
              <transition-stub
                name="expand-transition"
              >
                <!---->
              </transition-stub>
            </div>
          </div>
          <!---->
          <div
            class="text-center footer"
          >
            <a
              block=""
              class="v-btn-stub"
              color="white"
              data-toggle-expanded-btn=""
            >
      古い算定情報（障害・居宅介護）を表示
              <v-icon-stub
                right=""
              >
                M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z
              </v-icon-stub>
            </a>
          </div>
        </div>
        <div
          class="z-visiting-care-for-pwsd-calc-specs-card root"
          data-z-data-accordion=""
        >
          <z-subheader-stub>
            算定情報（障害・重度訪問介護）
          </z-subheader-stub>
          <div
            class="v-item-group theme--light v-expansion-panels"
          >
            <div
              aria-expanded="false"
              class="v-expansion-panel panel"
              data-expansion-panel=""
            >
              <button
                class="v-expansion-panel-header header"
                data-expansion-panel-header=""
                type="button"
              >
                <div
                  class="item"
                >
                  <div
                    class="text-caption text--secondary"
                  >
            適用期間
                  </div>
                  <div
                    class="text-body-2 text--primary"
                  >
                    <time
                      datetime="2008-04-16"
                    >
                      平成20年4月16日
                    </time>
                    <span>
                      〜
                    </span>
                    <time
                      datetime="2012-10-29"
                    >
                      平成24年10月29日
                    </time>
                  </div>
                </div>
                <transition-stub
                  name="fade-transition"
                >
                  <div
                    class="v-expansion-panel-header__icon"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-chevron-down theme--light"
                    />
                  </div>
                </transition-stub>
              </button>
              <transition-stub
                name="expand-transition"
              >
                <!---->
              </transition-stub>
            </div>
            <div
              aria-expanded="false"
              class="v-expansion-panel panel"
              data-expansion-panel=""
            >
              <button
                class="v-expansion-panel-header header"
                data-expansion-panel-header=""
                type="button"
              >
                <div
                  class="item"
                >
                  <div
                    class="text-caption text--secondary"
                  >
            適用期間
                  </div>
                  <div
                    class="text-body-2 text--primary"
                  >
                    <time
                      datetime="1972-05-03"
                    >
                      昭和47年5月3日
                    </time>
                    <span>
                      〜
                    </span>
                    <time
                      datetime="1989-04-02"
                    >
                      平成1年4月2日
                    </time>
                  </div>
                </div>
                <transition-stub
                  name="fade-transition"
                >
                  <div
                    class="v-expansion-panel-header__icon"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-chevron-down theme--light"
                    />
                  </div>
                </transition-stub>
              </button>
              <transition-stub
                name="expand-transition"
              >
                <!---->
              </transition-stub>
            </div>
          </div>
          <!---->
          <div
            class="text-center footer"
          >
            <a
              block=""
              class="v-btn-stub"
              color="white"
              data-toggle-expanded-btn=""
            >
      古い算定情報（障害・重度訪問介護）を表示
              <v-icon-stub
                right=""
              >
                M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z
              </v-icon-stub>
            </a>
          </div>
        </div>
        <div
          class="z-home-visit-long-term-care-calc-specs-card root"
          data-z-data-accordion=""
        >
          <z-subheader-stub>
            算定情報（介保・訪問介護）
          </z-subheader-stub>
          <div
            class="v-item-group theme--light v-expansion-panels"
          >
            <div
              aria-expanded="false"
              class="v-expansion-panel panel"
              data-expansion-panel=""
            >
              <button
                class="v-expansion-panel-header header"
                data-expansion-panel-header=""
                type="button"
              >
                <div
                  class="item"
                >
                  <div
                    class="text-caption text--secondary"
                  >
            適用期間
                  </div>
                  <div
                    class="text-body-2 text--primary"
                  >
                    <time
                      datetime="2008-04-16"
                    >
                      平成20年4月16日
                    </time>
                    <span>
                      〜
                    </span>
                    <time
                      datetime="2012-10-29"
                    >
                      平成24年10月29日
                    </time>
                  </div>
                </div>
                <transition-stub
                  name="fade-transition"
                >
                  <div
                    class="v-expansion-panel-header__icon"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-chevron-down theme--light"
                    />
                  </div>
                </transition-stub>
              </button>
              <transition-stub
                name="expand-transition"
              >
                <!---->
              </transition-stub>
            </div>
            <div
              aria-expanded="false"
              class="v-expansion-panel panel"
              data-expansion-panel=""
            >
              <button
                class="v-expansion-panel-header header"
                data-expansion-panel-header=""
                type="button"
              >
                <div
                  class="item"
                >
                  <div
                    class="text-caption text--secondary"
                  >
            適用期間
                  </div>
                  <div
                    class="text-body-2 text--primary"
                  >
                    <time
                      datetime="1972-05-03"
                    >
                      昭和47年5月3日
                    </time>
                    <span>
                      〜
                    </span>
                    <time
                      datetime="1989-04-02"
                    >
                      平成1年4月2日
                    </time>
                  </div>
                </div>
                <transition-stub
                  name="fade-transition"
                >
                  <div
                    class="v-expansion-panel-header__icon"
                  >
                    <i
                      aria-hidden="true"
                      class="v-icon notranslate mdi mdi-chevron-down theme--light"
                    />
                  </div>
                </transition-stub>
              </button>
              <transition-stub
                name="expand-transition"
              >
                <!---->
              </transition-stub>
            </div>
          </div>
          <!---->
          <div
            class="text-center footer"
          >
            <a
              block=""
              class="v-btn-stub"
              color="white"
              data-toggle-expanded-btn=""
            >
      古い算定情報（介保・訪問介護）を表示
              <v-icon-stub
                right=""
              >
                M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z
              </v-icon-stub>
            </a>
          </div>
        </div>
        <z-fab-speed-dial-stub
          data-fab=""
          icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
        >
          <z-fab-speed-dial-button-stub
            color="secondary"
            icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
            to="/offices/2/home-visit-long-term-care-calc-specs/new"
          >
          算定情報（介保・訪問介護）を登録
          </z-fab-speed-dial-button-stub>
          <z-fab-speed-dial-button-stub
            color="secondary"
            icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
            to="/offices/2/visiting-care-for-pwsd-calc-specs/new"
          >
          算定情報（障害・重度訪問介護）を登録
          </z-fab-speed-dial-button-stub>
          <z-fab-speed-dial-button-stub
            color="secondary"
            icon="M19,13H13V19H11V13H5V11H11V5H13V11H19V13Z"
            to="/offices/2/home-help-service-calc-specs/new"
          >
          算定情報（障害・居宅介護）を登録
          </z-fab-speed-dial-button-stub>
        </z-fab-speed-dial-stub>
      </v-tab-item-stub>
    </div>
  </div>
</div>
`;
